docs/_data/toc.yml

```diff
@@ -11,62 +11,61 @@
 
 - caption: shotgun
   children:
-  - page: /shotgun/
+  - page: shotgun
   - text: python-api
-    url: " https://developer.shotgunsoftware.com/python-api/"
+    url: "https://developer.shotgunsoftware.com/python-api/"
   - text: rest-api
-    url: " https://developer.shotgunsoftware.com/rest-api/"
+    url: "https://developer.shotgunsoftware.com/rest-api/"
   - text: event-triggers
     url: "https://github.com/shotgunsoftware/shotgunEvents/wiki"
   - text: jira-bridge
-    url: " https://developer.shotgunsoftware.com/sg-jira-bridge/"
-  - page: /shotgun/action_menu_items/
+    url: "https://developer.shotgunsoftware.com/sg-jira-bridge/"
+  - page: shotgun-action-menu-items
     children:
     - text: examples
-      url: " http://developer.shotgunsoftware.com/python-api/cookbook/examples/ami_handler.html"
+      url: "http://developer.shotgunsoftware.com/python-api/cookbook/examples/ami_handler.html"
     - text: reference
       url: "https://support.shotgunsoftware.com/entries/110709-How-to-create-custom-menu-items-for-integration-with-other-pipeline-tools"
 
 - caption: toolkit
   children:
-  - page: /toolkit/
-  - page: /toolkit/learning-resources/
+  - page: toolkit
+  - page: toolkit-learning-resources
     children:
-    - page: /toolkit/learning-resources/guides/
+    - page: toolkit-guides
       children:
-      - page: /toolkit/learning-resources/guides/advanced_config/
-      - page: /toolkit/learning-resources/guides/editing_app_setting/
-      - page: /toolkit/learning-resources/guides/installing_app/
-    - page: /toolkit/learning-resources/tutorial/
+      - page: toolkit-guides-advanced-config
+      - page: toolkit-guides-editing-config
+      - page: toolkit-guides-adding-app
+    - page: toolkit-pipeline-tutorial
     - text: webinars
       url: "https://support.shotgunsoftware.com/hc/en-us/articles/360002566514"
   - text: core-api
-    url: " https://developer.shotgunsoftware.com/tk-core/"
-  - page: /toolkit/frameworks/
+    url: "https://developer.shotgunsoftware.com/tk-core/"
+  - page: toolkit-frameworks
     children:
     - text: qt-widgets
-      url: " http://developer.shotgunsoftware.com/tk-framework-qtwidgets"
+      url: "http://developer.shotgunsoftware.com/tk-framework-qtwidgets"
     - text: shotgun-utils
-      url: " http://developer.shotgunsoftware.com/tk-framework-shotgunutils"
-  - page: /toolkit/app_apis/
+      url: "http://developer.shotgunsoftware.com/tk-framework-shotgunutils"
+  - page: toolkit-app-apis
     children:
     - text: publish-api
-      url: " https://developer.shotgunsoftware.com/tk-multi-publish2/"
+      url: "https://developer.shotgunsoftware.com/tk-multi-publish2/"
   - text: app-development
     url: "https://support.shotgunsoftware.com/entries/95440137"
-  - page: /toolkit/tutorial/
 
 - caption: rv
   children:
-  - page: /rv/
+  - page: rv
```

Comment:
how about something more descriptivel, like `rv-home` or `rv-index`?

Reply:
Updated all index pagenames to end in `-index`